dashboard/app/helpers/child_account_helper.rb

```diff
@@ -6,9 +6,12 @@ module ChildAccountHelper
   def parental_permission_banner_data(user, request)
     student_lockout_date = Policies::ChildAccount.lockout_date(user, approximate: true)
 
-    student_lockout_date ||= Cpa::ALL_USER_LOCKOUT_DATE if request.params.key?(:show_parental_permission_banner)
-
-    return unless student_lockout_date
+    if request.params.key?(:show_parental_permission_banner)
+      student_lockout_date ||= Cpa::ALL_USER_LOCKOUT_DATE
+    else
+      return unless student_lockout_date
+      return unless DCDO.get('cap_student_warnings_enabled', false)
```

Comment:
```suggestion
      return unless DCDO.get('cap_student_warnings_enabled', !Rails.env.production?)
```
I wonder if it would be a good idea to set defaults this way so dev, test, and staging have the features running by default.

Reply:
I also thought about this, but I wasn't sure if we want this